java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -925,7 +931,10 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
             0L, // xgroupDelConsumer(streamKey1, groupName1, consumer1)
             true, // xgroupDestroy(streamKey1, groupName1)
             true, // xgroupDestroy(streamKey1, groupName2)
-            1L, // .xdel(streamKey1, new String[] {"0-1", "0-5"});
+            1L, // .xdel(streamKey1, new String[] {"0-1", "0-5"})
+            "1-0", // xadd(streamKey3, Map.of("f0", "v0"), id("1-0"))
+            OK, // xgroupCreate(streamKey3, groupName3, "0")
+            new Map[] {}, // xinfoGroups(streamKey3)
```

Comment:
Why it is empty?

Reply:
That's the same test for python. https://github.com/aws/glide-for-redis/pull/1753